tests/helpers/test_entity_component.py

```diff
@@ -338,7 +353,8 @@ def platform_setup(hass, config, add_entities, discovery_info=None):
     assert p_hass is hass
     assert p_entry is entry
 
-    assert component._platforms[entry.entry_id].scan_interval == timedelta(seconds=5)
+    assert component._platforms[entry.entry_id].scan_interval == timedelta(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4603109)